apps/condo/domains/property/schema/Property.js

```diff
@@ -77,9 +88,50 @@ const Property = new GQLListSchema('Property', {
                 },
             },
         },
+        flatsCount: {
```

Comment:
untsCount
Term `Flat` is applicable only for Property.type == building. And there are no flats in the Property.type == village.

Reply:
renamed to unitsCount